pahfit/component_models.py

```diff
@@ -134,3 +134,37 @@ def evaluate(x, tau, x_0, fwhm):
             profile = Drude1D(amplitude=1.0, fwhm=fwhm, x_0=x_0)
             tau_x = tau * profile(x)
             return (1.0 - np.exp(-1.0 * tau_x)) / tau_x
+
+
+class AreaGaussian1D(Fittable1DModel):
+
```

Comment:
Please add a docstring to provide basic information about this function.  Like what currently exists for `S07_attenuation` for example.